codelists/models.py

```diff
@@ -130,6 +130,42 @@ def latest_version(self):
         return self.versions.filter(draft_owner__isnull=True).order_by("id").last()
 
 
+class Handle(models.Model):
+    codelist = models.ForeignKey(
+        "Codelist", on_delete=models.CASCADE, related_name="handles"
+    )
+    name = models.CharField(max_length=255)
+    slug = models.SlugField()
+    organisation = models.ForeignKey(
+        "opencodelists.Organisation",
+        null=True,
+        related_name="handles",
+        on_delete=models.CASCADE,
+    )
+    user = models.ForeignKey(
+        "opencodelists.User",
+        null=True,
+        related_name="handles",
+        on_delete=models.CASCADE,
+    )
+    is_current = models.BooleanField()
```

Comment:
Should this have a default or `null=True` set?

Reply:
No, because I want to always be set explicitly.